From 0fff09b4f33eb1320e4124d93a3207dcd3acd417 Mon Sep 17 00:00:00 2001
From: Jan Kiszka <jan.kiszka@siemens.com>
Date: Fri, 1 Jan 2021 14:54:51 +0100
Subject: [PATCH] sdkchroot: Drop bogus restriction to x86 hosts

This check is not just unneeded, at least for arm64. It also breaks the
build on non-x86 hosts because it takes away a dependency that the image
class requires.

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
---
 meta/recipes-devtools/sdkchroot/sdkchroot.bb | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/meta/recipes-devtools/sdkchroot/sdkchroot.bb b/meta/recipes-devtools/sdkchroot/sdkchroot.bb
index d7a848e2..e367eae2 100644
--- a/meta/recipes-devtools/sdkchroot/sdkchroot.bb
+++ b/meta/recipes-devtools/sdkchroot/sdkchroot.bb
@@ -35,12 +35,6 @@ ROOTFS_FEATURES += "clean-package-cache generate-manifest export-dpkg-status"
 ROOTFS_MANIFEST_DEPLOY_DIR = "${DEPLOY_DIR_SDKCHROOT}"
 ROOTFS_DPKGSTATUS_DEPLOY_DIR = "${DEPLOY_DIR_SDKCHROOT}"
 
-python() {
-    if d.getVar("HOST_ARCH") not in ['i386', 'amd64']:
-        raise bb.parse.SkipRecipe("SDK doesn't support {} as host".format(
-            d.getVar("ROOTFS_ARCH")))
-}
-
 SDK_PREINSTALL += " \
     debhelper \
     autotools-dev \
-- 
2.26.2

